_includes/footer-site-slim.html

```diff
@@ -3,79 +3,59 @@
-
-                    <a href="https://libguides.mit.edu/mit-feeds" title="RSS">
-                        <svg class="icon-social--rss" width="2048" height="2048" viewBox="-320 -384 2048 2048" xmlns="http://www.w3.org/2000/svg"><g transform="scale(1 -1) translate(0 -1280)"><path d="M384 192q0 -80 -56 -136t-136 -56t-136 56t-56 136t56 136t136 56t136 -56t56 -136zM896 69q2 -28 -17 -48q-18 -21 -47 -21h-135q-25 0 -43 16.5t-20 41.5q-22 229 -184.5 391.5t-391.5 184.5q-25 2 -41.5 20t-16.5 43v135q0 29 21 47q17 17 43 17h5q160 -13 306 -80.5 t259 -181.5q114 -113 181.5 -259t80.5 -306zM1408 67q2 -27 -18 -47q-18 -20 -46 -20h-143q-26 0 -44.5 17.5t-19.5 42.5q-12 215 -101 408.5t-231.5 336t-336 231.5t-408.5 102q-25 1 -42.5 19.5t-17.5 43.5v143q0 28 20 46q18 18 44 18h3q262 -13 501.5 -120t425.5 -294 q187 -186 294 -425.5t120 -501.5z" fill="black"></path></g></svg>
-                        <span class="sr">RSS</span>
-                    </a><!-- End RSS -->
-                </div><!-- end .social -->
-                <div class="wrap-middle">
-                    <div class="wrap-sitemap">
-                        <nav class="sitemap-libraries-abbrev" aria-label="MIT Libraries site menu">
-                            <h2 class="sr">MIT Libraries navigation</h2>
-                            <a class="item" href="https://libraries.mit.edu/search">Search</a>
-                            <a class="item" href="https://libraries.mit.edu/hours">Hours &amp; locations</a>
-                            <a class="item" href="https://libraries.mit.edu/borrow">Borrow &amp; request</a>
-                            <a class="item" href="https://libraries.mit.edu/research-support">Research support</a>
-                            <a class="item" href="https://libraries.mit.edu/about">About us</a>
-                        </nav>
-                    </div><!-- end .links-all -->
-                    <div class="wrap-policies">
-                        <nav aria-label="MIT Libraries policy menu">
-                            <span class="item"><a href="https://libraries.mit.edu/privacy" class="link-sub">Privacy</a></span>
-                            <span class="item"><a href="https://libraries.mit.edu/permissions" class="link-sub">Permissions</a></span>
-                            <span class="item"><a href="https://libraries.mit.edu/accessibility" class="link-sub">Accessibility</a></span>
-                        </nav>
-                    </div>
-                </div>
-            </div><!-- end .identity -->
+      <div class="footer-main" aria-label="MIT Libraries footer">
+        <div class="identity">
+          <div class="wrap-logo-lib">
+            <a href="https://libraries.mit.edu" class="logo-mit-lib" alt="MIT Libraries Logo">
+              <span class="sr">MIT Libraries home</span>
+              <img
+                src="https://cdn.dev1.mitlibrary.net/files/branding/local/mitlib-wordmark.svg"
+                alt="MIT Libraries logo"
+                height="38"
```

Comment:
I believe we'd agreed on 35px. Ultimately, I don't really care, but I want to make it's consistent. I asked in #engx what everyone is using to confirm.